mccpp/box/box.hpp

```diff
@@ -0,0 +1,21 @@
+
+#pragma once
+
+#include <iostream>
+#include <Eigen/Dense>
+#include <vector>
+#include <math.h>
+
+class Box
+{
+  public:
+     std::vector<double> box_dims;
+   Eigen::MatrixXd coordinates;
+
+  double volume(void);
+
+  Eigen::MatrixXd coord_wrap(void);
+
+  std::vector<double> minimum_image_dist(std::vector<double> i_particle);
```

Comment:
You should use an index here. That is what the rest of the code supports now anyways. This will ultimately be faster too.